src/main/java/dev/latvian/mods/kubejs/block/callback/EntityInsideBlockCallback.java

```diff
@@ -0,0 +1,13 @@
+package dev.latvian.mods.kubejs.block.callback;
+
+import net.minecraft.core.BlockPos;
+import net.minecraft.world.entity.Entity;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.state.BlockState;
+
+public class EntityInsideBlockCallback extends EntitySteppedOnBlockCallback {
```

Comment:
I don't like this inheritance tree. It makes no sense conceptually, and if Mojang add a new parameter to stepped on then it will need reworking. Maybe instead rename EntitySteppedOnBlockCallback to a more generic 'EntityBlockCallback' and use that directly in both places?